### PR TITLE
Add persistence instructions

### PR DESCRIPTION
**Pull request by gabriel-farache**
https://issues.redhat.com/browse/FLPATH-1178

Add persistence instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,14 @@ The repository includes a variety of serverless workflows, such as:
...
 ### Pre-requisites
-o utilize the workflows contained in this repository, the Orchestrator Deployment must be installed on your OpenShift Container Platform (OCP) cluster. For detailed instructions on installing the Orchestrator, please visit the [Orchestrator Helm Repository](https://www.parodos.dev/orchestrator-helm-chart/)
+Utilize the workflows contained in this repository, the Orchestrator Deployment must be installed on your OpenShift Container Platform (OCP) cluster. For detailed instructions on installing the Orchestrator, please visit the [Orchestrator Helm Repository](https://www.parodos.dev/orchestrator-helm-chart/)
+
+### Persistence pre-requisites
+If persistence is enbaled, you must have a PostgreSQL instance running in the cluster, in the same `namespace` as the workflows.
+
+A `secret` containing the instance credentials must exists as well. 
+
+See https://www.parodos.dev/orchestrator-helm-chart/postgresql on how to install a PostgreSQL instance using helm. In this document, a `secret` holding the credentials is created.
```
Comment (masayag) [inline README.md:21]: quick note, but let not change this text: there are 2 methods to deploy DB: once with helm and another by opensfhit template.

File: greeting/README.md [hunk 2]
```diff
@@ -7,12 +7,46 @@ Its purpose is to demonstrate functionality of the workflows system.
...
+      databaseSchema: greeting
+```
+
+Make sure the above values match what is deployed on your namespace.
+
+You can patch the resource by running (update it if needed with your own values):
+```bash
+  oc patch sonataflow/greeting \
+    --type merge \
+    -p '{"spec: {"persistence":{"postgresql":{"secretRef":{"name":"sonataflow-psql-postgresql","userKey":"postgres-username","passwordKey":"postgres-password"},"serviceRef":{"name":"sonataflow-psql-postgresql","port":5432,"databaseName":"sonataflow","databaseSchema":"greeting"}}}}}'
```
Comment (masayag) [inline greeting/README.md:47]: is there an option to indent this json blob?

Comment (gabriel-farache) [reply greeting/README.md:47]: no, I like big fat inlined JSON, don't you?

Comment (gabriel-farache) [reply greeting/README.md]: done :)

File: README.md (hunk 1 above)
Comment (gabriel-farache) [reply README.md]: I made it clearer to follow the helm section

**Review by masayag — APPROVED** [review]

**Pull request merged**